client/my-sites/theme/main.jsx

```diff
@@ -89,7 +89,8 @@ const ThemeSheet = React.createClass( {
 	},
 
 	onButtonClick() {
-		this.props.defaultOption.action( this.props );
+		const { defaultOption } = this.props;
+		defaultOption.action && defaultOption.action( this.props );
```

Comment:
> The check for `defaultOption.action` here implies that sometimes a button click would do nothing. That's probably not the case, so is the check necessary?

Yeah, `action` can be null. But that doesn't mean a button click would do nothing, since I'm also [passing](https://github.com/Automattic/wp-calypso/pull/6844/files#diff-f576b3c13daa17f9ccee514c4c46bc07R302) the `href` prop to the button component.

It's in line with the rest of this PR -- a couple of options now only have a `getUrl` (and no `action`) attr. We use them to populate either `href`s or `onClick`s respectively (since at the end of the day, all affected components are just `a`s in a nice wrapping).

BTW we still tap into `action` for options that only have a `getUrl` to e.g. add tracking for the 'more' button menu, which is guaranteed as `onClick` is always run before the `href` is followed.